Makefile

```diff
@@ -5,9 +5,19 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 # Running in Dapper
 
-BINARIES := bin/lighthouse-agent bin/lighthouse-coredns
+gotodockerarch = $(patsubst arm,arm/v7,$(1))
+dockertogoarch = $(patsubst arm/v7,arm,$(1))
```

Comment:
This is already defined is Submariner's `Makefile` and I suspect would be needed in other projects (operator?)

Perhaps it's best to defined these functions in `Makefile.inc`?

Reply:
Yes, but I needed them here to have a working PR ;-). I’ll move everything appropriate to Shipyard once this gets merged.